src/commons/BrowserUtils.ts

```diff
@@ -352,11 +352,32 @@ export namespace BrowserUtils {
     isExist(selector);
 
     Reporter.debug(`Switching to an Iframe by selector '${selector}'`);
-    tryBlock(
-      () => browser.switchToFrame($(selector)),
-      'Failed to switch frame'
-    );
+
     chillOut();
+
+    switch (browser.capabilities.browserName) {
+      case 'chrome': {
+        Reporter.debug('Case chrome');
+        tryBlock(
+          () => browser.switchToFrame($(selector)),
```

Comment:
I suggest to create iFrameSelector variable and set it's value inside the switch  and leave the browser.swichToFrame outside the switch block

Reply:
this way we will loose some of the logs info. also what type will be that variable?